composer.json

```diff
@@ -11,6 +11,8 @@
     ],
     "require": {
         "php": ">=7.1.0",
+        "ext-json": "*",
```

Comment:
I don't think we need to add `ext-json`, it's already required by Laravel.

Reply:
We do since we use `json_` methods in this library. It's always better to explicitly state your dependencies in your composer.json, even if it's used in Laravel.